pepper_bringup/launch/pepper_full_py.launch

```diff
@@ -3,6 +3,7 @@
   <arg name="nao_ip"            default="$(optenv NAO_IP 127.0.0.1)" />
   <arg name="nao_port"          default="$(optenv NAO_PORT 9559)" />
   <arg name="namespace"           default="pepper_robot" />
+  <arg name="use_odometry" default="True" />
```

Comment:
Thanks for pointing it out, it seems that the same comment made by @k-okada a few days ago doesn't show up on this PR so thanks for reposting it.